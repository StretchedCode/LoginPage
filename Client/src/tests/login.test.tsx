import Login from "../pages/login"
import { render, screen } from "@testing-library/react"
import userEvent from "@testing-library/user-event"

test("render of Form", () => {
  const { container } = render(<Login></Login>)

  expect(screen.getByRole("form")).toMatchSnapshot()
})

describe("Form testing", () => {
  test("Testing header label", () => {
    const { container } = render(<Login></Login>)

    expect(screen.getByRole("heading").textContent).toMatch("Log In.")
  })
})

test("DOM structure matches snapshot", () => {
  const { container } = render(<Login></Login>)

  expect(container).toMatchSnapshot()
})
