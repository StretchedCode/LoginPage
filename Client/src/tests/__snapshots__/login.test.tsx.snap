// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DOM structure matches snapshot 1`] = `
<div>
  <div
    role="label"
  >
    Bruh
  </div>
</div>
`;
